src/ol/source/imagestaticsource.js

```diff
@@ -58,7 +59,7 @@ goog.inherits(ol.source.ImageStatic, ol.source.Image);
  * @inheritDoc
  */
 ol.source.ImageStatic.prototype.getImage =
-    function(extent, resolution, pixelRatio, projection) {
+    function(extent, resolutions, pixelRatio, projection) {
```

Comment:
This method is [called by the layer renderer](https://github.com/openlayers/ol3/blob/10c715ad9acc93228bda5d146de6d9044c7f93c7/src/ol/renderer/canvas/canvasimagelayerrenderer.js#L117-L118) with the (singular) view resolution.  Changing the argument name to `resolutions` has no effect here, but it is also misleading (and I think it represents a misunderstanding).